### ISSUE TITLE
חיווי לשחקן

### ISSUE THREAD

**Issue by erelsgl** [Oimit/GameDev---GuitarHero#1]
ניסיתי לשחק, ולא היה ברור לי מתי אני עושה את הדבר הנכון. צריך לתת לשחקן חיווי כלשהו,
למשל, כשלוחצים על האות של הצבע הכחול, להגדיל קצת את הריבוע הכחול כדי שהשחקן יבין שהלחיצה שלו נקלטה.

**Comment by Oimit** [comment]
יש ניקוד שמראה שכאשר אתה לוחץ בזמן הנכון, עולה לך נקודה. במידה ולא לחצת
יורדת נקודה..

בתאריך יום ד׳, 10 בנוב׳ 2021 ב-20:06 מאת Erel Segal-Halevi <
***@***.***>:

**Comment by erelsgl** [comment]
נכון, רואים את הנקודות, אבל הניקוד הוא למעלה בצד שמאל, ולא בטוח שהשחקן יקשר
בין הניקוד הזה לבין מה שקורה למטה בצד ימין.
אפשר לעשות שכאשר לוחצים על z/x/c, העיגול בצבע המתאים יהבהב (יגדל ויקטן),
ואם בדיוק פוגע בו ריבוע בצבע המתאים, אז הוא יגדל עוד יותר. או לחשוב על
חיווי אחר כלשהו.

**Comment by Oimit** [comment]
הבנתי, יבוצע ויתוקן :)
בתאריך יום ד׳, 10 בנוב׳ 2021 ב-20:16 מאת Erel Segal-Halevi <
***@***.***>:
